src/bindgen/ir/annotation.rs

```diff
@@ -53,6 +55,27 @@ impl AnnotationSet {
         self.must_use && config.language != Language::Cython
     }
 
+    pub(crate) fn deprecated_note(&self, config: &Config) -> Option<String> {
```

Comment:
Why cloning? Can't you return an `Option<&str>`?

Reply:
I can, I made it return Option<&str> instead.